packages/db/postgres/functions.sql

```diff
@@ -325,6 +325,19 @@ BEGIN
 END
 $$;
 
+CREATE OR REPLACE FUNCTION uploads_by_type(query_type TEXT) RETURNS TEXT
```

Comment:
I'm wondering why do we need a procedure here? Shouldn't we just use client count API?
I see that's how it is done for other metrics, but I'm not sure if/why it's needed?

I wonder if was done like this because of the Supabase client issue loading the all table even when just "counting"? 
But that can be workarounded by adding `range(0, 1)`

Probably one for @vasco-santos

Reply:
For `sum` we can't do with the client, and that was the main motivation to not use the client. The `getPinStatusMetrics`  using `pin_from_status_total` I don't recall if there was a reason. Either the cast to pin status, or was just added here with the `sums` and we didn't consider using the client. 

We use the client for other metrics counts, so if we can use it here, I think we should

Reply:
@alexandrastoica did you find a reason to not have it using supabase client?

Reply:
Will change to client! :)